cmd/util.go

```diff
@@ -48,6 +44,9 @@ import (
 	"github.com/pulumi/pulumi/pkg/util/contract"
 	"github.com/pulumi/pulumi/pkg/util/gitutil"
 	"github.com/pulumi/pulumi/pkg/workspace"
+	survey "gopkg.in/AlecAivazis/survey.v1"
```

Comment:
Yeah this was done by VS Code automatically. I'll revert this change. I kept it as-is thinking it would be needed since the formatter applied it. I'll find the setting that might have caused this.

Reply:
Just checked and it seems VS Code's default Go code formatter is `goreturns`. I changed it to `gofmt` and it stopped reorganizing the package imports on save.